src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+public class FindCommand extends Command {
+
+    private String find;
```

Comment:
would it be better to name this string keyword instead of find to prevent confusion with the find function in TaskList